src/main/java/com/codecool/virtualpub/data/GameData.java

```diff
@@ -0,0 +1,78 @@
+package com.codecool.virtualpub.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum GameData {
+    DATA(
+            new String[]{
+                    "Alexander", "Benjamin", "Charlotte", "Daniel", "Emma",
+                    "Fiona", "Gabriel", "Hannah", "Isaac", "Julia",
+                    "Kevin", "Lucy", "Matthew", "Nora", "Oliver",
+                    "Penelope", "Quinn", "Ryan", "Sophia", "Thomas",
+                    "Abigail", "Brian", "Chloe", "David", "Evelyn",
+                    "Grace", "Henry", "Isabella", "Jack", "Katherine",
+                    "Liam", "Mia", "Nathan", "Olivia", "Peter",
+                    "Rebecca", "Samuel", "Tyler", "Victoria", "William",
+                    "Zoe", "Avery", "Blake", "Caleb", "Delilah",
+                    "Elijah", "Faith", "George", "Ivy", "Jackson"
+            },
+            List.of(
+                    // WINE drinks
+                    new Drink("Tokaji Aszú", DrinkType.WINE, 1990),
```

Comment:
Using an enum for the `GameData` is not the right tool and it's confusing. (Are there going to be other variants than DATA?). I suggest simply making the fields `public static final`. Like this:

```java

public class GameData {

    public static final String[] customerNames = new String[]{
        //...
    };

    public static final List<Drink> drinks = List.of(
        //...
    );
}

```